modules/label-data-fields-bias-monitoring.adoc

```diff
@@ -0,0 +1,14 @@
+:_module-type: PROCEDURE
+
+[id="label-data-fields-bias-monitoring_{context}"]
+= Labeling data fields
+
+The models have not yet provided appropriate field names for our inputs and outputs (all some form of `customer_data+input-x`). You can apply a set of name mappings to these to apply meaningful names to the fields. This is done by sending a POST request to the `/info/names` endpoint:
+
+Run the following command to apply the `/info/names` endpoint:
+[source]
+----
+./2-BiasMonitoring./scripts/apply_name_mapping.sh
+----
+
+To understand how the payload is structured, see link:https://github.com/trustyai-explainability/odh-trustyai-demos/blob/main/2-BiasMonitoring/scripts/apply_name_mapping.sh[apply_name_mapping.sh].
```

Comment:
replace link with a reference to their local version
2-BiasMonitoring/scripts/apply_name_mapping.sh